letter-sender/src/test/java/com/christmas/letter/sender/controller/LetterControllerTest.java

```diff
@@ -0,0 +1,81 @@
+package com.christmas.letter.sender.controller;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.christmas.letter.sender.model.Letter;
+import com.christmas.letter.sender.model.Location;
+import com.christmas.letter.sender.service.LetterService;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+@WebMvcTest(LetterController.class)
+class LetterControllerTest {
+
+  @SuppressWarnings("SpringJavaInjectionPointsAutowiringInspection")
```

Comment:
Do we need this suppress annotation?

Reply:
yes because do not recognizet the @Autowired